client/v2/common/models/box.go

```diff
@@ -0,0 +1,10 @@
+package models
```

Comment:
I believe this is code generated for the Box endpoint, so it should be in the other PR.

Reply:
@algoidurovic Thanks for catching - addressed via https://github.com/algorand/go-algorand-sdk/pull/337/commits/309afb3e783ab799085c3037e6c5c406f808ed92.